feature-utils/poly-import/test/storage.test.js

```diff
@@ -0,0 +1,26 @@
+import { MockPod } from "@polypoly-eu/api/dist/mock-pod";
+import { FeatureFileStorage } from "../src";
+
+describe("File storage ", () => {
+    let fileStorage;
+    beforeAll(() => {
+        const pod = new MockPod();
+        fileStorage = new FeatureFileStorage(pod);
+    });
+    it("can be instantiated", () => {
+        expect(fileStorage.files).toBeNull;
+    });
+});
+
+describe("Can remove archives ", () => {
```

Comment:
Looking at the code - it doesn't remove any archives, does it? Isn't it more `it("ignores non-existent archives")`? (I also think this is more of an `it` to "File storage" rather than its own `describe`, but that part is a nit.)

Reply:
It's quite incomplete at the moment, I'm afraid. But yes, this specific one could be renamed and instantiated as you say.